package.json

```diff
@@ -1,6 +1,90 @@
 {
+  "name": "@eyedea-sockets/syncano-socket-markdown-converter",
+  "version": "0.0.1",
+  "description": "Description",
+  "repository": "https://github.com/eyedea-io/syncano-socket-markdown-converter.git",
+  "public": true,
+  "author": "Eyedea <hello@eyedea.io> https://eyedea.io",
+  "license": "MIT",
+  "keywords": [
+    "syncano",
+    "socket"
+  ],
+  "peerDependencies": {
+    "@syncano/cli": "0.13.0-alpha.f6ea95d4"
```

Comment:
This should be:

```
"@syncano/cli": "^0.13.0-alpha.f6ea95d4"
```

Which means version `0.13.0-alpha.f6ea95d4` or higher `^`